### PR TITLE
#2323 What's New in 24.1 beta

### PR DESCRIPTION
**Pull request by micheleRP**
Fixes https://github.com/redpanda-data/documentation-private/issues/2323
[Preview](https://deploy-preview-403--redpanda-docs-preview.netlify.app/24.1/get-started/whats-new-241/)

### CODE REVIEW

File: modules/get-started/pages/whats-new-241.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+== New commands
+
+New commands and properties in this release include the following:
+
+- xref:reference:rpk/rpk-registry/rpk-registry.adoc[`rpk registry`]: This includes commands to interact with the Schema Registry.
+- xref:reference:rpk/rpk-transform/rpk-transform.adoc[`rpk transform`]: This includes commands to develop, deploy, and manage data transforms.
+
+== Next steps
+
+* xref:get-started:install-beta.adoc[]
```
Comment (Deflaimun) [inline modules/get-started/pages/whats-new-241.adoc:21]: Title will change to `Install 24.1 beta` once we merge https://github.com/redpanda-data/docs/pull/402

**Review by Deflaimun — COMMENTED** [review]
lgtm from tools perspective

**Review by Deflaimun — APPROVED** [review]

File: modules/get-started/pages/whats-new-241.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+= What's New in 24.1 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 24.1 beta. 
+
+== Write caching
+
+xref:develop:config-topics.adoc#configure-write-caching[Write caching] is a relaxed mode of `acks=all` that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. For clusters in development mode, write caching is enabled by default. For clusters in production mode, it is disabled by default.
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new-241.adoc:8]: We should link to the docs showing where to configure it on topics

File: modules/get-started/pages/whats-new-241.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+== RBAC
+xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). It allows you to create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those authorizations. Using RBAC gives you a more flexible tool to manage authorization for your users, especially when you have complex organizational hierarchies, large numbers of users, or both.
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new-241.adoc]: 'allows you to create roles, assign roles to multiple users, and assign those roles as the authorized or denied party for an ACL' 

@deniscoady @graphcareful  is this the best way to say this?

Comment (mattschumpert) [reply modules/get-started/pages/whats-new-241.adoc]: @micheleRP would say 'flexible and efficient' (more than just 'flexible')

File: modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -1,6 +1,6 @@
 * xref:home:index.adoc[Docs Home]
 * xref:get-started:index.adoc[Get Started]
-** xref:get-started:whats-new.adoc[]
+** xref:get-started:whats-new-241.adoc[]
```
Comment (mattschumpert) [inline modules/ROOT/nav.adoc:3]: @micheleRP I guess we are missing topic-aware leader balancing

**Review by mattschumpert — APPROVED** [review]

File: modules/ROOT/nav.adoc (hunk 4 above)
Comment (micheleRP) [reply modules/ROOT/nav.adoc:3]: @mattschumpert correct, we were not planning doc for that for beta. See [here](https://redpandadata.atlassian.net/wiki/spaces/DOC/pages/531234904/Core+Version+24.1).

Comment (mattschumpert) [reply modules/ROOT/nav.adoc:3]: All good @micheleRP. ship it!

File: modules/get-started/pages/whats-new-241.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/get-started/pages/whats-new-241.adoc:8]: @mattschumpert We had to merge this, but I'll open a new PR in a minute to add the changes to `rpk security acl` and any other feedback. For this comment: isn't this link at the beginning of the blurb the correct link for configuring write caching?

**Pull request merged**